api/internal/localizer/locloader.go

```diff
@@ -117,7 +117,8 @@ func (ll *locLoader) Load(path string) ([]byte, error) {
 // Otherwise, New returns an error.
 func (ll *locLoader) New(path string) (ifc.Loader, error) {
 	repoSpec, err := git.NewRepoSpecFromURL(path)
-	if err == nil && repoSpec.Ref == "" {
+	isRemote := err == nil
```

Comment:
nit: I think it will be more readable if we rename the above err to something like `fetchRemoteErr` and then checking if that is nil or not in the if conditions below, rather than creating a new bool here